commitizen/commands/commit.py

```diff
@@ -73,10 +73,24 @@ def prompt_commit_questions(self) -> str:
         return message
 
     def __call__(self):
+        signoff: bool = (
+            self.arguments.get("signoff") or self.config.settings["always_signoff"]
+        )
         dry_run: bool = self.arguments.get("dry_run")
         write_message_to_file: bool = self.arguments.get("write_message_to_file")
-
+        retry: bool = self.arguments.get("retry")
+        no_retry: bool = self.arguments.get("no_retry")
+        retry_after_failure: bool = self.config.settings.get("retry_after_failure")
```

Comment:
Should we add a default value to these `get`?

Reply:
Hmmm, we can add a explicit default value but `.get` has a default value to `None` if not specified.

Reply:
Yep, but `None` is not a `bool`, `False` would be better 🙂